Tasks/Common/webdeployment-common/webconfigutil.ts

```diff
@@ -44,6 +44,11 @@ function addMissingParametersValue(appType: string, webConfigParameters) {
         },
         'Go': {
             'GoExeFilePath': ''
+        },
+        'java_springboot': {
+            'JAVA_PATH' : '%JAVA_HOME%\\bin\\java.exe',
+            'JAR_PATH' : '',
```

Comment:
Should we have some default value for JAR_PATH?

Reply:
jar path is mandatory field so we can't pre-populate any default value.